.asf.yaml

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+github:
+  enabled_merge_buttons:
+    squash: true
+    merge: false
+    rebase: false
+notifications:
+  commits: commits@yetus.apache.org
+  issues: notifications@yetus.apache.org
+  pullrequests: notifications@yetus.apache.org
```

Comment:
I think most of this goes to gitbox@ currently.

Reply:
Oh, I didn't know that ML.  I'll update the PR.

We need to update the web site in a separate issue: http://yetus.apache.org/mailinglists/

Reply:
We sent gitbox stuff there but the JIRA and other ASF stuff goes to those MLs to make things a bit less noisy. We were one of the first projects to go to the gitbox stuff before this config file existed.